queries.sql

```diff
@@ -0,0 +1,10 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-12-31' AND '2019-01-01';
```

Comment:
- [ ] The range here should be from January 1st 2016 to December 31st 2019. Kindly correct that 😄